test/Utilities/Microsoft.Testing.TestInfrastructure/TargetFrameworks.cs

```diff
@@ -18,6 +18,8 @@ public static class TargetFrameworks
 
     public static TestArgumentsEntry<string> NetCurrent { get; } = Net[0];
 
+    public static TestArgumentsEntry<string>[] NetLatest { get; } = new[] { Net[0] };
```

Comment:
why you need this one?

Reply:
I am using it for NativeAot tests, the other entry is not usable with ArguementsProvider because it is not an array so then the source gen of internal framework does not work. And I don't want to run native aot tests x times, or having to go and update them to run on latest when latest .net version changes.

Reply:
NetCurrent does not work work with the ArgumentsProvider because it is not an array. If I change it I would have to fix all the other tests using it, and I don't want to mix that into my PR.

Reply:
But you need an array if you test only 1 tfm? this is not clear..

Reply:
The current source gen will not generate correct code if this is not an array. Even for 1 item.